### ISSUE TITLE
Any plans for angular 2 module

### ISSUE THREAD

**Issue by shadeeka** [mikemclin/angular-acl#21]
This is really useful module, any plans for implement this using angular 2

**Comment by mikemclin** [comment]
Yes, but not necessarily in the near future, unless I get a pull request.  I haven't been able to spend a lot of time with Angular 2 yet.

I've actually never used this library in a project before.  I'll finally be implementing it into a project this week.  I imagine there will be some new tweaks and enhancements in the upcoming month.

I've always considered this project to be in BETA.  After I am actually using it in a project, I'll most likely tag the first `1.0` version.  At that point, I'll probably look into starting a `2.0` branch which would be for Angular 2 releases.

**Comment by scheung38** [comment]
Thanks @mikemclin that will be cool to use in Angular2
